core/pom.xml

```diff
@@ -81,6 +81,13 @@
             <version>${json.org.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>info.debatty</groupId>
+            <artifactId>java-string-similarity</artifactId>
```

Comment:
Ah, ok. I thought JaroWinkler is for the cases from real world always an improvement and we should replace the current approach (?)